debian/marquee/01-certbot

```diff
@@ -2,6 +2,6 @@
 
 apt install certbot python-certbot-apache
```

Comment:
Drop python-certbot-apache. Also rebase and  `rm -rf debian/marquee/apache/`, some things are still left.